csi/node_server.go

```diff
@@ -812,11 +812,51 @@ func getNodeServiceCapabilities(cs []csi.NodeServiceCapability_RPC_Type) []*csi.
 	return nscs
 }
 
+func (ns *NodeServer) requireHostNamespaceMounter(volume *longhornclient.Volume, volumeCapability *csi.VolumeCapability) (bool, error) {
+	if !requiresSharedAccess(volume, volumeCapability) {
+		return false, nil
+	}
+	if volume.Migratable {
```

Comment:
Is this covered in requiresSharedAccess?

Reply:
I think no, ther are 2 types of RWX:
1. Regular (might need to enter host namespace if RWX storage network is not enabled)
2. Migratable volume for Havester (no need to enter host namespace ever) 

WDYT?

Reply:
I see. Thank you.